applications/call_inspector/src/parsers/ci_parser_kamailio.erl

```diff
@@ -282,7 +282,7 @@ cleanse_data_and_get_timestamp(Data0) ->
 -spec cleanse_data_fold({ne_binary() | kz_now()} | ne_binary()
                        ,cleanse_acc()
                        ) -> cleanse_acc().
-cleanse_data_fold({RawTimestamp}, {Acc, TS}) ->
+cleanse_data_fold({'timestamp', RawTimestamp}, {Acc, TS}) ->
```

Comment:
Well, yes and no. It would have required a lot more effort to figure out if the variable inside the `{...}` was a list or not. This was an easier fix, and tagged tuples are more idiomatic, I believe. Why wrap a timestamp in a 1-tuple to being with?